bibernate-core/src/main/java/com/bobocode/petros/bibernate/session/validation/EntityMappingValidator.java

```diff
@@ -1,15 +1,97 @@
 package com.bobocode.petros.bibernate.session.validation;
 
+import com.bobocode.petros.bibernate.exceptions.ValidationException;
+import com.bobocode.petros.bibernate.scanner.EntityScanner;
+import com.bobocode.petros.bibernate.utils.EntityUtils;
+import lombok.SneakyThrows;
 import lombok.experimental.UtilityClass;
 
 import javax.sql.DataSource;
-import java.util.Collections;
+import java.lang.reflect.Field;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.List;
 import java.util.Set;
+import java.util.function.Predicate;
 
 @UtilityClass
 public class EntityMappingValidator {
+    private static final String TABLE_NAME_LABEL = "TABLE_NAME";
+    private static final String COLUMN_NAME_LABEL = "COLUMN_NAME";
+
+    private static final String TABLE_DOESNT_EXIST_MSG = "Table '%s' doesn't exist.";
+    private static final String FIELD_DOESNT_EXIST_MSG = "Field '%s', doesn't exist in table '%s'";
 
     public Set<MappingViolationResult> validate(DataSource dataSource, Set<String> entityPackages) {
```

Comment:
Could you please add javadoc to your code?

Reply:
Done